ifrs17-template/Files/DataNodes/DataNodes_CH.csv

```diff
@@ -3,7 +3,7 @@ ReportingNode,,,,,,
 CH,,,,,,
 ,,,,,,
 @@InsurancePortfolio,,,,,,
-SystemName,DisplayName,ContractualCurrency,LineOfBusiness,ValuationApproach,OciType,
+SystemName,DisplayName,ContractualCurrency,LineOfBusiness,ValuationApproach,OciType,Scenario,
```

Comment:
remove Scenario from Data Node csv files, it should be optional not mandatory. Apply also for all recurrences.